docs/install/install-server/install-server.inc

```diff
@@ -64,25 +64,52 @@ Install the software
         First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
         Next install inmanta with pip install in the newly created virtual env.
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
             # Install python3 >= 3.6 and git
             sudo python3 -m venv /opt/inmanta
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
+            
+            # Install PostgreSQL
+            sudo apt-get install postgresql postgresql-client
+
+
+        Download the configuration file named ``inmanta.cfg`` (this name is arbitrary) in your virtual env:
+
+        .. code-block:: sh
+
+            wget -O /opt/inmanta/inmanta.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/inmanta.cfg"
+
+
+        If you want to use the dashboard you need to install it as well. There are two available options:
 
+        1. Get the pre-built package from `our github page <https://github.com/inmanta/inmanta-dashboard/packages/81295>`_ on the right side and under ``Assets``, you will see the compressed package, download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under ``[dashboard]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
 
-        The misc folder in the source distribution contains systemd service files for both the server and the agent. Also
-        install ``inmanta.cfg`` from the misc folder in ``/etc/inmanta/inmanta.cfg``
+        .. code-block:: bash
 
-        If you want to use the dashboard you need to install it as well. Get the source from
-        `our github page <https://github.com/inmanta/inmanta-dashboard/releases>`_ Next, build and install the dashboard. For
-        this you need to have yarn and grunt:
+            path=/opt/inmanta/package/dist
+
+
+        Then the dashboard can be started using below command:
+
+        .. code-block:: bash
+
+            inmanta -vv -c /opt/inmanta/inmanta.cfg server
+
+        OR
+
+        2. Get the source from `our github page <https://github.com/inmanta/inmanta-dashboard/releases>`_ on the right and under `Assets`, you will see the source. Next, build and install the dashboard. For this you need to have yarn and grunt:
```

Comment:
I would remove this part, using the pre-built npm is easier

Reply:
@bart: wdyt?

Reply:
How should I go about it? keep it or remove it?

Reply:
use the prebuilt version